pkgs/development/interpreters/python/build-python-package-common.nix

```diff
@@ -29,4 +29,8 @@ attrs // {
 
     runHook postInstall
   '';
+
+  fixupPhase = attrs.fixupPhase or ''
+    rm -rf $out/bin/__pycache__
```

Comment:
Bytecode produces by 2.x and 3.x is different.

```
$ ls -Al $(nix-build -A python2.pkgs.docutils)/bin
total 132
-r-xr-xr-x 1 root root  483 Jan  1  1970 rst2html5.py
-r--r--r-- 1 root root  745 Jan  1  1970 rst2html5.pyc
-r-xr-xr-x 1 root root 1620 Jan  1  1970 .rst2html5.py-wrapped
-r-xr-xr-x 1 root root  482 Jan  1  1970 rst2html.py
-r--r--r-- 1 root root  619 Jan  1  1970 rst2html.pyc
-r-xr-xr-x 1 root root 1073 Jan  1  1970 .rst2html.py-wrapped
-r-xr-xr-x 1 root root  483 Jan  1  1970 rst2latex.py
-r--r--r-- 1 root root  749 Jan  1  1970 rst2latex.pyc
...
```
```
$ ls -Al $(nix-build -A python3.pkgs.docutils)/bin
total 92
dr-xr-xr-x 2 root root 4096 Jan  1  1970 __pycache__
-r-xr-xr-x 1 root root  483 Jan  1  1970 rst2html5.py
-r-xr-xr-x 1 root root 1621 Jan  1  1970 .rst2html5.py-wrapped
-r-xr-xr-x 1 root root  482 Jan  1  1970 rst2html.py
-r-xr-xr-x 1 root root 1074 Jan  1  1970 .rst2html.py-wrapped
-r-xr-xr-x 1 root root  483 Jan  1  1970 rst2latex.py
...
```
Both cases need to be handled.

Then, there is the matter of where to put this function. Whoever uses `buildPythonPackage`, should be able to use this. Even with `format = "other";`. Therefore, I suggest putting this in the `postFixup` of `mk-python-derivation.nix`.